core/templates/dev/head/services/contextual/UrlService.js

```diff
@@ -19,9 +19,16 @@
 
 oppia.factory('UrlService', ['$window', function($window) {
   return {
+    // This function is for testing purposes (to mock $window.location)
+    queryUrlParams: function() {
```

Comment:
getQueryUrlParams. Always start method names with verbs.

That said -- this doesn't seem to be returning queryUrlParams. Shouldn't it be more like getCurrentLocation()?

Please be careful to be precise when naming things. This is important to get right.

Reply:
Ok, done